wcon_schema.json

```diff
@@ -29,7 +29,7 @@
                 ]}
             },
             "additionalProperties": false,
-            "required": ["this", "prev", "next"]
```

Comment:
Agreed, I should fix the spec language to be consistent.  The principle here is "don't make people include stuff with no information content", so I think that requiring only `"this"` is the right policy